books/free-programming-books-it.md

```diff
@@ -242,7 +242,7 @@
 
 * [Il manuale di riferimento di Python](http://docs.python.it/html/ref/)
 * [Il tutorial di Python](http://docs.python.it/html/tut/)
-* [Immersione in Python 3](http://gpiancastelli.altervista.org/dip3-it/) - Mark Pilgrim, _traduzione di: Giulio Piancastelli_ (HTML) [(PDF)](http://gpiancastelli.altervista.org/dip3-it/d/diveintopython3-it-pdf-latest.zip)
```

Comment:
role notation

Reply:
I'll homogenize it across files with `trad.:`

Reply:
Recovered at 0a4dbc179e4b2f6346fc15579d7c5edec1b3433e